### PR TITLE
Adding comment about minimum supported Orion server version.

### PR DESCRIPTION
**Pull request by dturchin**
@dshush Please review. Adding comment about Orion server version support.

### CODE REVIEW

**Comment by dshush** [comment]
@dturchin I've made some structure changes - take a look and tell me if it's

**Comment by dturchin** [comment]
@dshush Good idea! Thanks. I'll merge the pull request.

File: _integrations/solarwinds.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ draft: false
 type: System Monitoring
 ---
 
+#### Compatibility
```
Comment (noamshemesh) [inline _integrations/solarwinds.md]: It will be step 2 in the tutorial... and it's not a real step

Comment (dshush) [inline _integrations/solarwinds.md]: True, but we don't have a way to add something that is not a step in the tutorial..

**Pull request merged**